contracts/SubscriptionBilling.sol

```diff
@@ -113,17 +111,19 @@ contract SubscriptionBilling is HasNoContracts {
         require(DateTime.toTimestamp(currentYear, currentMonth) >= DateTime.toTimestamp(_year, _month),
             "Can't distribute future fees!");
 
+        address[] memory members = federation.getMembers();
```

Comment:
Should we have some sort of default behavior here? What happens if federation.getMembers() stops working? We don't have an "update federation contract" behavior here. 
So I think there's a case where money gets locked in, in this contract. right?

Reply:
Yes, but at the moment it kind of can't happen. Either `federation` is `selfdestruct` (which we don't allow) or `federation.getMembers()` returns an empty array, which also don't support at the moment. What would you suggest?